src/main/java/nextstep/subway/favorite/dto/FavoriteResponse.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.subway.favorite.dto;
+
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.station.dto.StationResponse;
+
+public class FavoriteResponse {
+    private Long id;
+    StationResponse source;
+    StationResponse target;
```

Comment:
```suggestion
    private StationResponse source;
    private StationResponse target;
```

같은 실수가 반복되는 느낌입니다. 
조금만 더 신경을 써주시면...

Reply:
😭